my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
comment on the comment: backticks ("\`") can be used to denote code, e.g. `from matplotlib import pyplot as plt`